docs/040_pepr-tutorials/030_create-pepr-operator.md

```diff
@@ -798,7 +791,7 @@ Make sure Pepr is update to date
 npx pepr update
 ```
 
-Build the Pepr manifests
+Build the Pepr manifests (Already built with appropriate RBAC)
```

Comment:
What was already built with appropriate RBAC?

Reply:
Yes, with `npx pepr build --rbac-mode=scoped` and then adding by hand the missing elements.